src/Applications/LDAS_App/process_rst.csh

```diff
@@ -79,55 +79,34 @@ if ( -e /etc/os-release ) then
 else
   module load other/nco-4.6.8-gcc-5.3-sp3 
 endif
-#setenv OMPI_MCA_shmem_mmap_enable_nfs_warning 0
-#setenv MKL_CBWR SSE4_2 # ensure zero-diff across archs
-#setenv MV2_ON_DEMAND_THRESHOLD 8192 # MVAPICH2
+
 setenv LAIFILE `find ${BCSDIR}/lai_clim*`
 setenv LD_LIBRARY_PATH ${LD_LIBRARY_PATH}:${BASEDIR}/Linux/lib
 
 limit stacksize unlimited
  
-$INSTDIR/bin/esma_mpirun -np 56 bin/mk_GEOSldasRestarts.x -a ${SPONSORID} -b ${BCSDIR} -t ${TILFILE} -m ${MODEL} -s ${SURFLAY} -j Y
+$INSTDIR/bin/esma_mpirun -np 56 bin/mk_GEOSldasRestarts -a ${SPONSORID} -b ${BCSDIR} -t ${TILFILE} -m ${MODEL} -s ${SURFLAY} -j Y
 
 sleep 3
 
-if($LSM_CHOICE == 1) then
-   /bin/cp OutData1/catch_internal_rst OutData2/catch_internal_rst
-else
-   /bin/cp OutData1/catchcn_internal_rst OutData2/catchcn_internal_rst
-endif
-
-$INSTDIR/bin/esma_mpirun -np 56 bin/mk_GEOSldasRestarts.x -a ${SPONSORID} -b ${BCSDIR} -t ${TILFILE} -m ${MODEL} -s ${SURFLAY} -j Y
+/bin/cp InData/${MODEL}_internal_rst OutData/${MODEL}_internal_rst
 
-_EOI_
+$INSTDIR/bin/esma_mpirun -np 56 bin/mk_GEOSldasRestarts -a ${SPONSORID} -b ${BCSDIR} -t ${TILFILE} -m ${MODEL} -s ${SURFLAY} -j Y
 
-    if($LSM_CHOICE == 1) sed -i '$ a\bin/Scale_Catch OutData1/catch_internal_rst OutData2/catch_internal_rst catch_internal_rst $SURFLAY $WEMIN_IN $WEMIN_OUT \'  mkLDASsa.j
-    if($LSM_CHOICE == 2) sed -i '$ a\bin/Scale_CatchCN OutData1/catchcn_internal_rst OutData2/catchcn_internal_rst catchcn_internal_rst $SURFLAY $WEMIN_IN $WEMIN_OUT \'  mkLDASsa.j
+${SCALE} InData/${models}_internal_rst OutData/${models}_internal_rst ${models}_internal_rst $SURFLAY $WEMIN_IN $WEMIN_OUT 
 
-    sed -i '$ a\ \'  mkLDASsa.j
-    sed -i '$ a\## Done creating catch*_internal_rst file \'  mkLDASsa.j
-    sed -i '$ a\ \'  mkLDASsa.j
+# Done creating catch*_internal_rst file
 
-    cat << _EOI2_ > mkLDASsa.j2
 sleep 2
 
-if($LSM_CHOICE == 1) then
-   if (-f irrigation_internal_rst && $RUN_IRRIG == 1) then 
-      ncks -4  -v IRRIGFRAC,PADDYFRAC,LAIMIN,LAIMAX,CLMPT,CLMST,CLMPF,CLMSF irrigation_internal_rst -A catch_internal_rst
-   endif
-   ln -s  catch_internal_rst catch_internal_rst.$YYYYMMDD
-else
-   if (-f irrigation_internal_rst && $RUN_IRRIG == 1) then 
-      ncks -4 -v IRRIGFRAC,PADDYFRAC,LAIMIN,LAIMAX,CLMPT,CLMST,CLMPF,CLMSF irrigation_internal_rst -A catchcn_internal_rst 
-   endif
-   ln -s  catchcn_internal_rst catchcn_internal_rst.$YYYYMMDD
+if (-f irrigation_internal_rst && $RUN_IRRIG == 1) then 
```

Comment:
Why does this PR deal with irrigation?  Shouldn't the irrigation changes be in a separate GEOSldas PR that goes with the irrigation PR in GEOSgcm_GridComp? 
The same comment applies elsewhere in this script.

Reply:
My question : what is this irrigation stuff doing in this script in the first place.

 ncks -4 -v IRRIGFRAC,PADDYFRAC,LAIMIN,LAIMAX,CLMPT,CLMST,CLMPF,CLMSF irrigation_internal_rst
 
 You all had merged an non-existing, never-tested irrigation model to develop many years ago. 
 I just deleted, because no BCs directory supports this. This was based on a temperory branch I created for Manuela many years ago, that should never have made anywhere in to a main repo. (It was in cvs)

Reply:
FYI, there is also extraneous irrigation code in the related PRs for the GEOSgcm_App and GMAO_Shared.  You can look up for yourself who committed these changes.  
cc: @gmao-jkolassa @weiyuan-jiang @biljanaorescanin

Reply:
I first checked whether you had merged anything related to the never-tested or never-ran version of irrigation model to the AGCM scripts. Looks like, AGCM side is safe. If you see otherwise, please delete, that version of irrigation model practically never existed.

route_internal stuff in regrid.pl: They were there since 2016 or so,  you were running old LDASsa at that time. Yes I made changes, not with this PR, many years ago when I added mk_RouteRestarts.F90 to mk_restarts.

irrigation_internal_rst infor in regrid.pl:
Yes I added and the reasons I will email you and Randy rather than posting here.
 If you decide to delete, it would be your responsibility to

Reply:
Removed the obsolete irrigation-related code from process_rst.csh